koji_containerbuild/plugins/hub_containerbuild.py

```diff
@@ -53,6 +64,33 @@ def _get_task_opts_and_opts(opts, priority, channel):
 
     return opts, taskOpts
 
+
+def _check_priority_allowlist(target, priority):
+    CONFIG = koji.read_config_files([(CONFIG_FILE, True)])
+    priority_target_list = CONFIG.get('priority_allowlist', 'priority_target_list_url')
+    req = Request(priority_target_list)
+    try:
+        response = urlopen(req)
+        prio_target_list = yaml.safe_load(response.read().decode())
+        if target in prio_target_list['targets']:
+            if not priority:
+                priority = koji.PRIO_DEFAULT - 1
+            else:
+                priority -= 1
+    except Exception as exc:
+        logger.warning('Failed to fetch priority target list: "%s":', priority_target_list)
+        if hasattr(exc, 'reason'):
+            if isinstance(exc.reason, socket.timeout):
```

Comment:
is this really `isinstance(exc.reason ...)` and not just `isinstance(exc, ...)`?

Reply:
Truly it is. Tested with `timeout = 0.01` in Jupyter.